a1/a1/bestmlp.py

```diff
@@ -0,0 +1,26 @@
+from sklearn.neural_network import MLPClassifier
+from sklearn.model_selection import GridSearchCV
+from a1 import util
+
+# Grid search hyperparameters
+# I'm guessing it has to do with the combination of all 3 parameters but sometimes 
+# we get "ConvergenceWarning: <ACTIVATION_TYPE>: Maximum iterations (200) reached and the optimization hasn't converged yet."
+param_grid = {
+    'activation' : ['logistic', 'tanh', 'relu', 'identity'], # Requirement (Note: logistic == sigmoid)
```

Comment:
I'm guessgin logistic is the same as sigmoid?

Reply:
yessirrr apprently so